### PR TITLE
Developer can now run lint and tests

### PR DESCRIPTION
**Pull request by cbeauchesne**
Here it is. @mfournier, @asaunier, @brunobesson, I'll need your help and comments on this PR.

What to do ? 

* Test it. See new README.md for instructions
* Does the README.md is crystal clear?
* Check that I didn't broke any hidden process ? (anonymisation scripts? build scripts ?)
* Is there some easy simplification to do? The actual status is pretty messy, lot of useless conf files, and data duplications ...
* I did not succeed to run `create_user_db_test.sh` in `dev_db.Dockerfile`. In consequence, you have an extra step before running tests. If one of you could help on this, it would be great ! :pray: 

Thank you for your help !

### CODE REVIEW

**Comment by cbeauchesne** [comment]
typo in READme.md : https://localhost:6543

File: README.md [hunk 1]
```diff
@@ -1,186 +1,50 @@
...
-    USER=something scripts/create_user_db.sh
+First, inititate test database : 
 
-To set up ElasticSearch
-----------------------
+```
+# Build test DB
+docker-compose up -d
+docker-compose exec postgresql /v6_api/scripts/create_user_db_test.sh
+docker-compose stop
```
[the remark this answers is not in the document]
Comment (cbeauchesne) [reply README.md]: The README is not clear enough.

Yes, the container must be running. This point must be explained as shortly as possible to a person that does not understand what is a container...

File: README_OLD.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
+API Application for camptocamp.org v6
```
[the remark this answers is not in the document]
Comment (cbeauchesne) [reply README_OLD.md]: Oups, typo, I should not have commited it : I've plan to move it in the wiki. Is it ok for you ?

File: README.md [hunk 3]
```diff
@@ -1,186 +1,48 @@
...
+### Run
 
-    git clone https://github.com/c2corg/v6_api.git
+```
+docker-compose up
+```
 
-Build
------
+:heart: [http://localhost:8080](http://localhost:6543) :heart:
```
[the remark this answers is not in the document]
Comment (asaunier) [reply README.md]: Or simply
```suggestion
:heart: http://localhost:6543 :heart:
```

File: README_OLD.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (asaunier) [reply README_OLD.md]: OK but then adding a link to the README would make the wiki page more visible.

File: README.md [hunk 4]
```diff
@@ -1,186 +1,48 @@
...
-To set up the database
-----------------------
+### Check code quality
 
-    scripts/create_user_db.sh
+In another terminal (`docker-compose up` must be running) :
 
-If you want to specify a specific string as user, you can instead use:
+```
+docker-compose exec api make -f config/docker-dev lint
```
Comment (asaunier) [inline README.md]: This indeed works and displays a list of errors if any. But no feedback is provided if no error is found: one might think nothing happened. Displaying an "Code quality check OK" would help.

File: README_OLD.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (cbeauchesne) [reply README_OLD.md]: done

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (cbeauchesne) [reply README.md]: done

File: README.md (hunk 4 above)
Comment (cbeauchesne) [reply README.md]: done

File: README.md [hunk 5]
```diff
@@ -1,186 +1,51 @@
...
+### Run
 
-    git clone https://github.com/c2corg/v6_api.git
+```sh
+docker-compose up
+```
 
-Build
------
+:heart: http://localhost:6543 :heart:
```
Comment (asaunier) [inline README.md]: The codacy extension recommands to use brackets instead of a raw URL

File: README.md [hunk 6]
```diff
@@ -1,186 +1,51 @@
...
-
-In case of unexpected exceptions, an error handler makes sure that a JSON
-response is returned. During development it can be convenient to see
-the Pyramid debugger toolbar in case of errors. To disable the error handler,
-add the following line to your config `config/{user}`:
-
-    export show_debugger_for_errors = true
+## Useful links in [wiki](https://github.com/c2corg/v6_api/wiki)
 
+* [Full info about development environment](https://github.com/c2corg/v6_api/wiki/Development-environment-on-Linux)
```
Comment (asaunier) [inline README.md]: Codacy recommends 2-space indentations for bullet list

**Comment by cbeauchesne** [comment]
Yep saw it. I did not understand the recommendation for the bullet list, so I removed it

**Comment by asaunier** [comment]
I am not sure either, I guess you had to add 2 spaces before the bullet sign ``*``?
No big deal :)

**Pull request merged**